configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
A user could set this case, but arguably maybe we wouldn't want to support it because it just defaults to the global value.  I'm ok either way here-- the key thing is that we should allow the `None` case for non-chunked behavior and allow a user-specified override corresponding to each analysis function.  Perhaps we don't need a global value case from config.default corresponding to `chunking=True`.